enginetest/script_queries.go

```diff
@@ -1100,6 +1100,33 @@ var ScriptTests = []ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "Issue #499",
```

Comment:
This tripped me up at first as I was getting different results when trying this locally on MySQL. I forgot that it interprets the given time on insertion as your local timezone, so setting my instance to UTC fixed it lol.